### PR TITLE
fix(data): Add Zishan-7

### PR DESCRIPTION
**Pull request by Zishan-7**

### CODE REVIEW

File: public/data/zishan.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/MohdZishan2539",
+        "icon": "twitter"
+      }
+    ]
+  }
+
```
Comment (krishguptadev) [inline public/data/zishan.json:18]:
```suggestion

```

File: public/data/zishan.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        "url": "https://github.com/Zishan-7",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/MohdZishan2539",
+        "icon": "twitter"
+      }
+    ]
+  }
```
Comment (krishguptadev) [inline public/data/zishan.json:17]:
```suggestion
}
```

File: public/data/zishan.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Zishan Mohd",
+    "bio": "I am Zishan Mohd, currently pursuing my Masters in Computer Apllication from Panjab University, Chandigarh. I am a newbie to open source. Looking forward to collaborate with awesome people.",
```
Comment (krishguptadev) [inline public/data/zishan.json:3]:
```suggestion
    "bio": "Masters in Computer Application Student, Panjab University, Chandigarh. Exploring open source. 😎",
```

**Review by krishguptadev — CHANGES_REQUESTED** [review]

File: public/data/zishan.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+        "url": "https://github.com/Zishan-7",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/MohdZishan2539",
+        "icon": "twitter"
+      }
+    ]
+  }
```
Comment (krishguptadev) [inline public/data/zishan.json:17]:
```suggestion
}

```

**Review by krishguptadev — CHANGES_REQUESTED** [review]

**Comment by krishguptadev** [comment]
Please do not just resolve this, but add this: https://github.com/EddieHubCommunity/LinkFree/pull/684#discussion_r753686071

Let us know if you need any help 🙂

**Comment by Zishan-7** [comment]
Thanks @krishguptadev I added your suggested changes.

**Review by krishguptadev — APPROVED** [review]

File: public/data/zishan.json [hunk 5]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Zishan Mohd",
+    "bio": "I am Zishan Mohd, currently pursuing my Masters in Computer Apllication from Panjab University, Chandigarh. I am a newbie to open source. Looking forward to collaborate with awesome people.",
```
Comment (Panquesito7) [inline public/data/zishan.json:3]:
```suggestion
    "bio": "I am Zishan Mohd, currently pursuing my Masters in Computer Application from Panjab University, Chandigarh. I am a newbie to open source. Looking forward to collaborating with awesome people!",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Pradumnasaraf — COMMENTED** [review]
LGTM, Some minor tweaks and spell errors.
See the above suggestions.👆

**Comment by Zishan-7** [comment]
Thanks for the suggestions!

**Review by krishguptadev — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Review by Vyvy-vi — COMMENTED** [review]
lgtm

**Pull request merged**